src/ReaderScreen/styles/commonStyle.js

```diff
@@ -0,0 +1,12 @@
+import { fontSizeForReader, fontColorForReader } from "../utils/util";
+import constant from "../../common/constant";
+
+const commonStyle = (header, isNightMode, type, fontSize, fontFace) => ({
```

Comment:
Make this a named export and import in index.js file in the same directory

Reply:
Fixed